modules/lang-painless/build.gradle

```diff
@@ -49,6 +49,17 @@ tasks.named("test").configure {
   jvmArgs '-XX:-OmitStackTraceInFastThrow', '-XX:-HeapDumpOnOutOfMemoryError'
 }
 
+tasks.named("yamlRestCompatTest").configure {
+    systemProperty 'tests.rest.blacklist', [
+            'painless/70_execute_painless_scripts/Execute with date field context (single-value)',
+            'painless/70_execute_painless_scripts/Execute with date field context (multi-value)',
+            'painless/70_execute_painless_scripts/Execute with geo point field context (single-value)',
+            'painless/70_execute_painless_scripts/Execute with geo point field context (multi-value)',
+            'painless/70_execute_painless_scripts/Execute with ip field context (single-value)',
+            'painless/70_execute_painless_scripts/Execute with ip field context (multi-value)'
+    ].join(',')
```

Comment:
is the blacklist taken from master or from 7.x? I guess this is temporary only until the changes are merged and we plan to backport to 7.13 too?

Reply:
right I am familiar with this dance, done it a few days ago, only instead of blacklisting I opened a PR against 7.x and skipped some tests temporarily. It was not clear to me which blacklist is used.